src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss

```diff
@@ -185,38 +185,48 @@ select[name="campaignId"] {
     font-size: 16px;
     font-weight: 500;
     line-height: 1.5;
-    padding: var(--givewp-spacing-3);
+    color: #1f2937;
```

Comment:
@jdghinson I'm guessing we just don't have a design system color we want to use here?

Reply:
Updated

Reply:
Nice! 🎉 

As a friendly note, @jdghinson, now that you're a dope dev-design unicorn, the convention we use is to include the hash in the follow up comment that is intended to resolve the comment. For example, for this comment you'd say something like:

Resolved in 0fc7b20

😃